style.css

```diff
@@ -296,3 +296,79 @@ li {
   color: #253858;
   margin-left: 20px;
 }
+
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  width: 100%;
+  background-image: url(/images/contact-form-m-bg.svg);
+  background-size: 50%;
+  background-position: right;
+  background-repeat: no-repeat;
+  padding-top: 50px;
+  border-bottom-left-radius: 0;
+}
+
+.contact-section h2 {
+  color: #fff;
+  width: 90%;
+  text-align: center;
+  padding-top: 25px;
+  font-weight: 700px;
+  font-size: 40px;
+}
+
+.contact-section p {
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-top: -10px;
+  padding: 15px;
+}
+
+.contact-section form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.contact-input {
+  padding: 15px;
+  width: 375px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  color: #172b4d;
+  outline: none;
+}
+
+.text-area {
+  height: 170px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  width: 375px;
+  color: #b3bac5;
+}
+
+.btn-contact {
```

Comment:
- [ ] Good job creating the contact section, you have done really well :clap: , however as mentioned in the project requirements we need to make sure we stick to the figma design as much as possible currently your design lcks the padding on left and right see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212894657-a14850d0-0aaa-4101-83e8-fbd0777feda2.png)

Figma dsign :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212894767-c33b6109-e549-4361-9c34-c81fe369078b.png)